### PR TITLE
Fixed HPC HP table bug which hid OC20 LR decay factor rule

### LINKED ISSUE #494: [HPC][OpenCatalyst] Missing HP rule for LR decay factor
The learning rate decay factor was mistakenly left out of the HP table for the OpenCatalyst benchmark, but is a very reasonable thing to tune and is usually allowed for other benchmarks.

I think we should allow to add a rule saying this parameter is unconstrained for HPC v2.0, despite being past the freeze deadline.

### PR DESCRIPTION
**Pull request by sparticlesteve**
A bug in the markdown from a missing HP table cell caused the oc20 `opt_learning_rate_decay_factor` rule being hidden when rendered.

This PR adds the cell which was missing on the above row, which was the text description of the `opt_learning_rate_decay_boundary_steps` parameter. Now the rules are correctly rendered in github.

Fixes #494

### CODE REVIEW

**Comment by sparticlesteve** [comment]
Approved in WG meeting on July 11, 2022.
